### DISCUSSION TITLE
Is there some widget like Material DropDownButtonFormfield?

### DISCUSSION THREAD

**Discussion by omensight** [bdlukaa/fluent_ui#881] [Q&A] [answered]
Hi, I'm looking for a form field TextBox widget, if there is something like that. Thanks in advance.

**Comment by bdlukaa** [comment, marked as answer]
You may be looking for `ComboBoxFormField` or `EditableComboBoxFormField`

**Reply by omensight** [reply]
This is exactly what I was looking for. Thanks
